governance-services/community/install/zip.md

```diff
@@ -23,19 +23,19 @@ The Governance Services Community Edition distribution zip contains the followin
 
 3. Copy the AMP files to the Alfresco `amps` and `amps_share` directories.
 
-    * Copy the `alfresco-governance-services-community-repo-23.1.0.255.amp` file to the Alfresco `amps` directory.
-    * Copy the `alfresco-governance-services-community-share-23.1.0.269.amp` file to the Alfresco `amps_share` directory.
+    * Copy the `alfresco-governance-services-community-repo-23.2.1.xxx.amp` file to the Alfresco `amps` directory.
+    * Copy the `alfresco-governance-services-community-share-23.2.1.xxx.amp` file to the Alfresco `amps_share` directory.
 
 4. Change into the root of the Alfresco Community Edition installation directory. Directories specified in the following procedures are relative to this directory.
 
 5. Run the following commands to install the AMP files:
 
     ```bash
-    java -jar bin\alfresco-mmt.jar install amps\alfresco-governance-services-community-repo-23.1.0.255.amp tomcat\webapps\alfresco.war
+    java -jar bin\alfresco-mmt.jar install amps\alfresco-governance-services-community-repo-23.2.1.xxx.amp tomcat\webapps\alfresco.war
     ```
 
     ```bash
-    java -jar bin\alfresco-mmt.jar install amps_share\alfresco-governance-services-community-share-23.1.0.269.amp tomcat\webapps\share.war
+    java -jar bin\alfresco-mmt.jar install amps_share\alfresco-governance-services-community-share-23.2.1.xxx.amp tomcat\webapps\share.war
```

Comment:
Add a hint about how customers should specify the exact released versions - the additional sentence should be at the same indent level as the `bash` commands:

```suggestion
    java -jar bin\alfresco-mmt.jar install amps_share\alfresco-governance-services-community-share-23.2.x.xxx.amp tomcat\webapps\share.war
```

    Replace `x.xxx` with the exact versions provided in the Governance Services Community Edition distribution zip.